WebDriverAgentLib/Categories/XCUIElement+FBWebDriverAttributes.m

```diff
@@ -25,23 +26,19 @@ @implementation XCUIElement (WebDriverAttributesForwarding)
 
 - (XCElementSnapshot *)fb_snapshotForAttributeName:(NSString *)name
 {
-  if (!self.exists) {
-    return [XCElementSnapshot new];
-  }
-
   // These attrbiutes are special, because we can only retrieve them from
   // the snapshot if we explicitly ask XCTest to include them into the query while taking it.
   // That is why fb_snapshotWithAllAttributes method must be used instead of the default fb_lastSnapshot
   // call
   if ([name isEqualToString:FBStringify(XCUIElement, isWDVisible)]) {
-    return ([self fb_snapshotWithAttributes:@[FB_XCAXAIsVisibleAttributeName]] ?: self.fb_lastSnapshot) ?: [XCElementSnapshot new];
```

Comment:
`self.lastSnapshot` can be nil, maybe we should call `fb_uniqueSnapshot` here instead?